src/PostgREST/Config.hs

```diff
@@ -198,16 +204,16 @@ readPathShowHelp = customExecParser parserPrefs opts
           |# raw-media-types="image/png, image/jpg"
           |
           |## logging level, the admitted values are: crit, error, warn and info.
-          |# log-level = "error"
+          |log-level = "error"
           |
           |## rollback all transactions by default, use for test environments
           |## disabled by default
-          |# tx-rollback-all = false
+          |tx-rollback-all = false
           |
           |## allow overriding the tx-rollback-all setting for a request by
           |## setting the Prefer: tx=[commit|rollback] header
           |## disabled by default
-          |# tx-allow-override = false
+          |tx-allow-override = false
```

Comment:
@wolfgangwalther Not sure if you'd be up for this. But recently we had feedback on hacker news about our config file: https://news.ycombinator.com/item?id=25170658

> I'd first assumed there would be a config file a mile long but it really is all Postgres.

So in general I think we should keep the number of options short(within reason). That lead me to think about your previous proposal about having a single `tx` config.

Would you be up for revisiting that? Would be another PR of course.

Reply:
Yeah sure, why not. I guess with the "In-Database configuration" stuff it would be even more impressive in terms of "size of config file needed".

Reply:
Cool. Then I think the config should be along the lines of your previous idea. Maybe like `tx-end=commit|rollback|commit-allow-override|rollback-allow-override`.

Reply:
See #1668